TiborKovari/brewery-app
# Feature/hero with image

Why was this change made?
The hero section was redesigned to visually stand out more, and a supporting image was added to enhance the user experience.

What does this change include?
Added a new image (beer.jpg) placed below the hero section.
Used TailwindCSS classes (w-full, object-cover, h-[calc(50vh)]) to ensure the image spans the full width and maintains proper proportions.
Updated the README.md file to improve project documentation.

How to test this change?
Run the application using npm run dev.
Open the application in the browser and verify:
The hero section is displayed correctly.
The image appears below the hero, spanning the full width and covering 70% of the screen height.

Is there anything to watch out for?
Ensure the image path works correctly across different environments (e.g., dev/prod).
This change does not affect other components or functionality.un dev and check if the image appears correctly below the hero.
